src/bitcoin-cli.cpp

```diff
@@ -928,6 +938,46 @@ static void GetProgressBar(double progress, std::string& progress_bar)
     }
 }
 
+/**
+ * GetFeerateBar constructs fee rate distribution bars using dynamic increment based on size.
+ *
+ * @param[in]  size      Size of fee rate group in bytes.
+ * @param[out] size_bar  String representation of the distribution bar.
+ * @param[in]  inc_size  Minimum size of fee rate groups in bytes.
+ * @param[in]  max_size  Maximum size of fee rate groups in bytes.
+ */
+static void GetFeerateBar(double size, std::string& size_bar, double inc_size, double max_size)
+{
+    if (size == 0) return;
+
+    static double INCREMENT{inc_size};
+    static const std::string BAR{"\u2591"};
+
+    if (max_size < 10000){
```

Comment:
Given that there so many very similar numbers differing in the number of zero digits, one may consider using `10'000` syntax to represent `10000` (https://en.cppreference.com/w/cpp/language/integer_literal). (I'm not sure if it is allowed in this repo though)

Reply:
Not sure about this as I have not seen such format for any constants in this repository.

Reply:
IDK, maybe @jonatack would know. 🙏